app/main.py

```diff
@@ -2,0 +1,53 @@
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
Similar to the `SoftwareEngineer` class, defining `skills` as a class attribute here will share it across all instances of `FrontendDeveloper`. Consider defining it as an instance attribute in the constructor.